lib/vsc/zk/rsync/source.py

```diff
@@ -238,39 +269,79 @@ def attempt_run(self, path, attempts=3):
             self.failed_queue.put(path)
             return 0, output  # otherwise client get stuck
         else:
-            self.log.debug('Still no destination after %s tries' % attempts)
+            self.log.warning('Still no destination after %s tries' % attempts)
             self.path_queue.put(path, priority=50)  # Keep path in queue
             self.path_queue.consume()  # But stop locking it
             time.sleep(self.TIME_OUT)  # Wait before new attempt
             return 1, None
 
-    def run_rsync(self, encpath, host, port):
+    def parse_output(self, output):
         """
-        Runs the rsync command with or without recursion, delete or dry-run option.
-        It uses the destination module linked with this session.
+        Parse the rsync output stats, and when verbose, print the files marked for transmission
+        """
+
+        if self.rsync_verbose:
+            outp = output.split("%s%s" % (os.linesep, os.linesep))
+            self.log.info('Verbose file list output is: %s' % outp[0])
```

Comment:
it is part of the stats yes, both total files in the dir as the transfered files